pkg/util/net.go

```diff
@@ -266,3 +275,74 @@ func GetHostIPv6(ipHint string) (string, error) {
 
 	return "", fmt.Errorf("unable to find host IPv6 address")
 }
+
+func FindDefaultRouteMinMTU() (mtu int, err error) {
+	ip_families := []int{netlink.FAMILY_V4, netlink.FAMILY_V6}
+
+	mtu_slice := []int{}
+
+	for _, ip_family := range ip_families {
+		new_mtu, err := FindDefaultRouteMTU(ip_family)
+		if err != nil {
+			continue
+		}
+		mtu_slice = append(mtu_slice, new_mtu)
+	}
+	if len(mtu_slice) > 0 {
+		return slices.Min(mtu_slice), nil
+	}
+	return 0, fmt.Errorf("could not find minimal MTU")
+}
+
+func FindDefaultRouteMTU(ip_family int) (mtu int, err error) {
+	link, err := FindDefaultRouteIface(ip_family)
+	if err != nil && link.MTU == 0 {
+		return 0, err
+	}
+
+	klog.Infof("using IP %d on Interface %s with MTU %d ", ip_family, link.Name, link.MTU)
```

Comment:
This log may be shown twice if using dual stack and its already printing in `getClusterMTU` function, isnt it?